app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,323 @@
+            android:text="@string/back_button_text"
+            android:textColor="@color/back_button_text_color" />
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/guideline_top"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            app:layout_constraintGuide_begin="224dp"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="12dp"
+            android:text="@string/age_limitation"
+            android:background="@drawable/age_limitation_background"
+            android:textColor="@color/white"
+            android:textSize="14sp"
+            android:textStyle="bold"
+            app:layout_constraintStart_toEndOf="@id/guideline_start"
+            app:layout_constraintTop_toBottomOf="@+id/guideline_top"
+            tools:ignore="MissingConstraints" />
+
+        <TextView
+            android:id="@+id/movie_title"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="38dp"
+            android:text="@string/movie_title_avengers_end_game"
+            android:textColor="@color/white"
+            android:textSize="44sp"
+            android:textStyle="bold"
+            app:layout_constraintStart_toEndOf="@id/guideline_start"
+            app:layout_constraintEnd_toStartOf="@id/guideline_end"
+            app:layout_constraintTop_toBottomOf="@+id/guideline_top"
+            tools:ignore="MissingConstraints" />
+
+        <TextView
+            android:id="@+id/tags"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:text="@string/movie_tags"
+            android:textColor="@color/second_main_color"
+            android:textSize="14sp"
+            app:layout_constraintStart_toEndOf="@id/guideline_start"
+            app:layout_constraintEnd_toStartOf="@id/guideline_end"
+            app:layout_constraintTop_toBottomOf="@id/movie_title" />
+
+        <ImageView
+            android:id="@+id/review_star1"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:src="@drawable/ic_star"
+            app:layout_constraintTop_toTopOf="@+id/number_of_reviews"
+            app:layout_constraintBottom_toBottomOf="@id/number_of_reviews"
+            app:layout_constraintStart_toEndOf="@id/guideline_start" />
+
+        <ImageView
+            android:id="@+id/review_star2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3dp"
+            android:src="@drawable/ic_star"
+            app:layout_constraintTop_toTopOf="@+id/number_of_reviews"
```

Comment:
app:layout_constraintTop_toTopOf="@+id/review_star1"
app:layout_constraintBottom_toBottomOf="@id/review_star1"

maybe for 2,3,4,5 start is better?